README.md

```diff
@@ -1,2 +1,18 @@
-# dfaker
-A generator of fake diabetes data.
+#Tidepool Dfaker 
+
+dfaker is a fake data generator that simulates blood glucose data over time. 
+**Note** This project is under active development 
+
+#Set up
+
+dfaker runs with python3. In addition, Python Anaconda is required. 
```

Comment:
I would soften this a bit to say something like = "Using the Anaconda scientific Python distribution is recommended as the easiest way of getting all project dependencies." It's not actually *required*... (And especially when we get the requirements.txt thing worked out, some people may opt for just installing everything via `pip`.)